### ISSUE TITLE
Custom embedding

### ISSUE THREAD

**Issue by raymondsim** [MaartenGr/BERTopic#1188]
Hi,

I currently have a pretrained encoder and decoder for text summarization (BART-based). And I want to use topic model as part of the loss function. I was wondering if the topic model supports custom embedding that I trained on my own? I saw the tutorial on Google colab uses TransformerDocumentEmbeddings from flair but not sure about using my own pre-trained model.

Thanks in advance!

**Comment by MaartenGr** [comment]
There is! You can either create a custom [backend](https://maartengr.github.io/BERTopic/getting_started/embeddings/embeddings.html#custom-backend) yourself or directly pass the [custom embeddings](https://maartengr.github.io/BERTopic/getting_started/embeddings/embeddings.html#custom-embeddings).
